packages/rum/src/boot/recorderApi.ts

```diff
@@ -62,12 +62,14 @@ export function makeRecorderApi(startRecordingImpl: StartRecording): RecorderApi
       session: RumSession,
       parentContexts: ParentContexts
     ) => {
-      lifeCycle.subscribe(LifeCycleEventType.SESSION_RENEWED, () => {
+      lifeCycle.subscribe(LifeCycleEventType.SESSION_EXPIRED, () => {
         if (state.status === RecorderStatus.Started) {
           stopStrategy()
           state = { status: RecorderStatus.IntentToStart }
```

Comment:
If a session expired, we might not need to restart it if it's now `LITE` no?

Reply:
I think what you're referring to is a few lines below in the `startStrategy`
https://github.com/DataDog/browser-sdk/blob/b38ae1d939b5d90d48595390843976be1c5d4871/packages/rum/src/boot/recorderApi.ts#L78-L82